x-pack/test/security_solution_endpoint/apps/integrations_feature_flag/artifact_entries_list.ts

```diff
@@ -0,0 +1,376 @@
+/*
```

Comment:
Copied this file from ../integrations. It is used in FF FTR config and once this feature is GA this file should be copied over to ../integrations.

Reply:
Yup, this needs to make it to our backlog cc @dasansol92.